content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -86,6 +86,7 @@ different Kubernetes components.
 | `CSIMigrationvSphere` | `false` | Beta | 1.19 | |
 | `CSIMigrationPortworx` | `false` | Alpha | 1.23 | |
 | `csiMigrationRBD` | `false` | Alpha | 1.23 | |
+| `CSINodeExpandSecret` | `false` | Alpha | 1.25 | |
```

Comment:
Later in this page, there's a list of feature gates and their descriptions. Please add a description there for `CSINodeExpandSecret`.

Reply:
sure, done.